pkg/factories/logger/logger.go

```diff
@@ -12,7 +12,7 @@ var loggerContext = "logger"
 func Get(ctx context.Context) logr.Logger {
 	l := ctx.Value(&loggerContext)
 	if l == nil {
-		return ctrl.Log
+		return ctrl.Log.WithName("")
```

Comment:
I don't understand this change

Reply:
https://github.com/go-logr/logr is upgrade. Logger change to struct from interface.